README.md

```diff
@@ -34,16 +34,26 @@ Jordan can also remind you of these things.
 7. discuss how crazy the media is in times of corona. 
 8. discuss how hard it is to be in self-isoltation.
 
+**Flow Chart**
```

Comment:
looks great! The blue is a bit dark for reading what's written inside the box though. Also I would implement the option to go back to choosing another topic for every time Jordan says or asks something in the conversation, so the arrow should go from maintain conversation to let participant pick a topic in my opinion.